README.md

```diff
@@ -172,6 +136,66 @@ The internal concurrency structure's setup has this value as a variable.
 The actual run time of the test can be longer than specified due to the
 responses delay. Use 0 for an infinite attack.
 
+#### `-format`
+Specifies the targets format to decode.
+
+##### `json` format
+
+The JSON format makes integration with programs that produce targets dynamically easier.
+Each target is one JSON object in its own line. If present, the body field must be base64 encoded.
+
+```bash
+jq -ncM '{method: "GET", url: "http://goku", body: "Punch!" | @base64, header: {"Content-Type": ["text/plain"]}}' |
+  vegeta attack -format=json -rate=100 | vegeta dump
+```
+
+##### `legacy` format
+
+The ill-defined legacy format almost resembles the plain-text HTTP message format but
```

Comment:
In here `ill-defined` seems self-deprecating, could be left out.

Reply:
OK, agree there's no need to deprecate it. I'll change the wording.